src/webapp-lib/app.pug

```diff
@@ -14,9 +14,11 @@ html(lang="en")
         img(src=require('!url-loader?mimetype=image/svg+xml!cocalc-icon-white-transparent.svg'))
       div.banner-error
         div.message
-          | Something bad must have happened.
+          | Timeout while loading CoCalc.
           br/
           | Try <a href="javascript:location.reload(true);">reloading the page</a> while holding your shift-key.
+          br/
+          | Or disable browser extensions blocking external sites, or try a different browser.
```

Comment:
I wonder if a link to a wiki page of troubleshooting techniques for the initial page load would be way more useful?

E.g., it could remark that IE11 is known broken *today*.

And it could be more clear, e.g., "whitelist cocalc in any ad blocking extensions..."

Reply:
In short, maybe replace all this with "It is taking a long time to load CoCalc, which is suspicious.  Please see these [troubleshooting ideas](...) or refresh your browser to try again."